app/models/mvi.rb

```diff
@@ -1,13 +1,16 @@
 # frozen_string_literal: true
 
-require 'mvi/responses/find_profile_response'
 require 'common/models/redis_store'
 require 'common/models/concerns/cache_aside'
+require 'master_veteran_index/orch_search_service'
+require 'master_veteran_index/responses/find_profile_response'
+require 'master_veteran_index/responses/add_person_response'
+require 'master_veteran_index/service'
 
 # Facade for MVI. User model delegates MVI correlation id and VA profile (golden record) methods to this class.
 # When a profile is requested from one of the delegates it is returned from either a cached response in Redis
 # or from the MVI SOAP service.
-class Mvi < Common::RedisStore
+class MVI < Common::RedisStore
```

Comment:
same comment as before -- this makes the class appear to be a constant, is it really necessary?